config/admission-controller/admission_controller.yaml

```diff
@@ -28,7 +28,4 @@ spec:
             requests:
               cpu: 20m
               memory: 20Mi
-          ports:
```

Comment:
why is this removed?

Reply:
Because port for webhook is already defined in `admission_controller_webhook_patch.yaml` and it generated this error on deployment
```
validatingwebhookconfiguration.admissionregistration.k8s.io/scylla-operator-autoscaler-validating-webhook-configuration configured
The Deployment "scylla-operator-autoscaler-admission-controller" is invalid: spec.template.spec.containers[0].ports[1].name: Duplicate value: "webhook"
```